### PR TITLE
[Metricbeat] Update MySQL dashboard

### PR DESCRIPTION
**Pull request by sayden**
## What does this PR do?
Update MySQL dashboard with the metrics merged in https://github.com/elastic/beats/pull/19844

## Related issues

- Implements https://github.com/elastic/beats/issues/16955

## Screenshots

image

### CODE REVIEW

File: metricbeat/module/mysql/_meta/kibana/7/dashboard/Metricbeat-mysql-overview.json [hunk 1]
```diff
@@ -1546,110 +1746,630 @@
...
+        "visState": {
+          "aggs": [],
+          "params": {
+            "axis_formatter": "number",
+            "axis_position": "left",
+            "axis_scale": "normal",
+            "default_index_pattern": "metricbeat-*",
+            "default_timefield": "@timestamp",
+            "id": "61ca57f0-469d-11e7-af02-69e470af7417",
+            "index_pattern": "",
```
Comment (kaiyan-sheng) [inline metricbeat/module/mysql/_meta/kibana/7/dashboard/Metricbeat-mysql-overview.json:2051]:
```suggestion
            "index_pattern": "metricbeat-*",
```

File: metricbeat/module/mysql/_meta/kibana/7/dashboard/Metricbeat-mysql-overview.json [hunk 2]
```diff
@@ -1546,110 +1746,630 @@
...
```
Comment (kaiyan-sheng) [inline metricbeat/module/mysql/_meta/kibana/7/dashboard/Metricbeat-mysql-overview.json:2182]:
```suggestion
            "index_pattern": "metricbeat-*",
```

File: metricbeat/module/mysql/_meta/kibana/7/dashboard/Metricbeat-mysql-overview.json [hunk 3]
```diff
@@ -34,7 +34,7 @@
             "panelIndex": "14",
             "panelRefName": "panel_0",
             "title": "Open Tables, Files, Streams",
-            "version": "7.3.1"
+            "version": "8.0.0-SNAPSHOT"
```
Comment (andresrc) [inline metricbeat/module/mysql/_meta/kibana/7/dashboard/Metricbeat-mysql-overview.json:37]: Are this versions significant? I mean, is it some kind of minimum required version? (ditto for the other changes below)

Comment (sayden) [reply metricbeat/module/mysql/_meta/kibana/7/dashboard/Metricbeat-mysql-overview.json:37]: Nope, this is the version that was used to create the dashboard / visualizations. Ideally Beats must use a released version and mot master, like I did in this PR by mistake. Maybe I should re-create it using a released version :thinking:

**Comment by kaiyan-sheng** [comment]
```
>> Dashboard format - module/mysql/_meta/kibana/7/dashboard/Metricbeat-mysql-overview.json:
   expected title with format 'Some title [Metricbeat Module]', found 'Open Tables Cache [MySQL Metricbeat] ECS': expected: 'Metricbeat', found: 'MySQL'
   expected title with format 'Some title [Metricbeat Module]', found 'Connection Errors [MySQL Metricbeat] ECS': expected: 'Metricbeat', found: 'MySQL'
   expected title with format 'Some title [Metricbeat Module]', found 'Commands Operations [MySQL Metricbeat] ECS': expected: 'Metricbeat', found: 'MySQL'
   expected title with format 'Some title [Metricbeat Module]', found 'SSL Cache [MySQL Metricbeat] ECS': expected: 'Metricbeat', found: 'MySQL'
```
So it should be [Metricbeat MySQL] instead of [MySQL Metricbeat]?

File: metricbeat/module/mysql/_meta/kibana/7/dashboard/Metricbeat-mysql-overview.json (hunk 3 above)
Comment (sayden) [reply metricbeat/module/mysql/_meta/kibana/7/dashboard/Metricbeat-mysql-overview.json]: I finally re-created the dashboard with 7.8.1

File: metricbeat/module/mysql/_meta/kibana/7/dashboard/Metricbeat-mysql-overview.json (hunk 2 above)
Comment (sayden) [reply metricbeat/module/mysql/_meta/kibana/7/dashboard/Metricbeat-mysql-overview.json]: Thanks! Fixed.

File: metricbeat/module/mysql/_meta/kibana/7/dashboard/Metricbeat-mysql-overview.json (hunk 1 above)
Comment (sayden) [reply metricbeat/module/mysql/_meta/kibana/7/dashboard/Metricbeat-mysql-overview.json]: Thanks! Fixed.

**Comment by sayden** [comment]
Ok, I have switched positions of `MySQL` and `Metricbeat` :nerd_face:

**Comment by sayden** [comment]
CI Error is related

**Pull request merged**